.gitmodules

```diff
@@ -18,16 +18,17 @@
 [submodule "submodules/cabal-helper"]
 	path = submodules/cabal-helper
 	# url = https://github.com/arbor/cabal-helper.git
-	url = https://github.com/alanz/cabal-helper.git
-	# url = https://github.com/DanielG/cabal-helper.git
+	# url = https://github.com/alanz/cabal-helper.git
+	url = https://github.com/DanielG/cabal-helper.git
+	# url = https://github.com/bubba/cabal-helper.git
 
 [submodule "submodules/ghc-mod"]
 	path = submodules/ghc-mod
 	# url = https://github.com/arbor/ghc-mod.git
-	url = https://github.com/alanz/ghc-mod.git
+	# url = https://github.com/bubba/ghc-mod.git
+	url = https://github.com/fendor/ghc-mod.git
```

Comment:
ghc-project-types, which depends on c-h and hie-plugin-api depends on https://github.com/mpickering/haskell-ide-engine/blob/hie-bios/hie-plugin-api/Haskell/Ide/Engine/Ghc.hs#L301 which is exposed in `ghc-project-types`. 
We need this ghc-mod dep, because c-h has a minor change in the API. The diff in `ghc-mod` has has like 3 changes, like, bumping c-h version, removing unknown type and adding instances for some type.

Reply:
yeah, I would love that! This would actually enable us to go to hackage, once c-h version 1.0 has been published!